doc/command_syntax_usage.rst

```diff
@@ -56,19 +56,19 @@ where ``hosts_file`` contains the ip address or ranges separated by newlines::
 ^^^^^^^^^
 Scanning
 ^^^^^^^^^
-The arguments required for a scan are the profile to use, the file path for the report
-and the facts to be collected. Optionally we can pass the number of Ansible forks.
-Finally an important argument is ``reset``. This tells rho that the profile you are
-passing in is either new or has been updated with changes in either the hosts or
-auths or both and that rho has to process it afresh.
+The arguments required for a scan are the profile to use, and the file path for
+the report. Optionally we can pass the number of Ansible forks and the facts to
+be collected. Finally the ``cache``argument tells rho that the profile you are
```

Comment:
Missing a space between the last ` and the word argument. Also I think we could replace argument to option since CLI calls it option.